perf/sds-tests/non-sds-1/httpbin.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Istio Authors
+#
```

Comment:
fix copyright year

Reply:
A lot is common between the two yaml files. 
One helm chart would do the trick with values option to enable sleep service.
How will you test crt rotation user load? I think better to use fortio instead of httpbin

Reply:
Done: fixed the copyright year, use a common helm chart with yaml files removed.
Cert rotation load is tested through creating a number of httpbin and sleep workloads (the number of workloads is an input parameter of the script). These workloads request for certificates at a customizable interval (e.g., every 1 minute), thereby creating the cert. rotation load. httpbin and sleepbin are used in the test to validate the mTLS. fortio is not used since the certificate rotation load is created by setting the number of workloads and the rotation interval, instead of sending TCP packets to a server through fortio.